### PR TITLE
[9.0] [MIG] account_due_list_aging_comments

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
Account Due List Aging Comments
===============================

This module extends the Payments and Due list to add the capability to
introduce aging comments in-line.

Usage
=====

To use this module, you need to go to:

* Accounting->Journal Entries->Payments and due list

### CODE REVIEW

File: account_due_list_aging_comments/views/payment_view.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
```
Comment (AaronHForgeFlow) [inline account_due_list_aging_comments/views/payment_view.xml]: \<openerp\> now is \<odoo\> and remove \<data\>

File: account_due_list_aging_comments/views/payment_view.xml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+            <field name="arch" type="xml">
+                <tree position="attributes">
+                    <attribute name="editable">top</attribute>
+                </tree>
+                <field name="reconciled" position="after">
+                    <field name="aging_comments"/>
+                </field>
+            </field>
+        </record>
+
```
Comment (AaronHForgeFlow) [inline account_due_list_aging_comments/views/payment_view.xml:18]: I would introduce the field in the form view also

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM despite minor comments :+1:

**Review by LoisRForgeFlow — APPROVED** [review]
tested on local :+1:

File: account_due_list_aging_comments/README.rst [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+This module extends the Payments and Due list to add the capability to
+introduce aging comments in-line.
+
+Usage
+=====
+
+To use this module, you need to go to:
+
+* Accounting->Journal Entries->Payments and due list
```
Comment (max3903) [inline account_due_list_aging_comments/README.rst]: Accounting -> Adviser ->Payments and due list

**Review by max3903 — CHANGES_REQUESTED** [review]

File: account_due_list_aging_comments/README.rst (hunk 3 above)
Comment (max3903) [reply account_due_list_aging_comments/README.rst]: @jbeficent `account_due_list` needs to depend on `account_accountant` to have access to the Adviser submenu (see #135)

File: account_due_list_aging_comments/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# © 2015 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    'name': "Payments Due list aging comments",
+    'version': '9.0.0.2.0',
```
Comment (lasley) [inline account_due_list_aging_comments/__openerp__.py]: 9.0.1.0.0

File: account_due_list_aging_comments/README.rst (hunk 3 above)
Comment (JordiBForgeFlow) [reply account_due_list_aging_comments/README.rst]: Done!

File: account_due_list_aging_comments/__openerp__.py (hunk 4 above)
Comment (JordiBForgeFlow) [reply account_due_list_aging_comments/__openerp__.py]: Done!

**Comment by JordiBForgeFlow** [comment]
@max3903 @lasley your review comments have been attended. Do you agree to merge this one now?

File: account_due_list_aging_comments/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# © 2015 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    'name': "Payments Due list aging comments",
+    'version': '9.0.0.1.0',
+    'category': 'Generic Modules/Payment',
+    'author': 'Odoo Community Association (OCA),'
+              'Eficent',
```
Comment (lasley) [inline account_due_list_aging_comments/__openerp__.py]: I would recommend to put yourself first here (instead of OCA).

**Review by lasley — APPROVED** [review]
LGTM - one minor thing I probably should have noticed the first time around, but nothing blocking

**Review by max3903 — APPROVED** [review]

**Pull request merged**